tests/validate.js

```diff
@@ -194,13 +194,94 @@ tap.test('manifest.css - empty array', (t) => {
 // .proxy
 //
 
-tap.test('manifest.proxy - empty object', (t) => {
-    t.notOk(proxy({}).error, 'should not return error');
+tap.test('manifest.proxy - empty array', (t) => {
+    t.notOk(proxy([]).error, 'should not return error');
```

Comment:
Are we sure we should support empty arrays as a value? Any reason to not give it a min length of 1 if its defined at all?

Reply:
Empty arrays is pretty fine. If its empty it will just not loop and do anything of the logic being done on any of the objects in the array.